workflow/workflow.sh

```diff
@@ -53,28 +53,39 @@ EOT
         echo -n "${command_names[idx]}"
         
         # Add spacing to separate names and descriptions in two columns
-        local name_length=${#command_names[idx]}
-        local col=$(expr $max_name_length - $name_length + 8)
+        local command_name="${command_names[idx]}"
+        local name_length=${#command_name}
+        local col_space=8
+        local col=$(($max_name_length - $name_length + $col_space))
         for (( i=1; i<=col; i++)); do echo -n ' '; done
         
         echo "${command_descriptions[idx]}"
     done
     
-    # Extra newline
-    echo ""
+    echo # Extra newline
 }
 
 
 # --------------------------------
 # Basic commands
 # --------------------------------
 
+command_names+=("self-installed")
+command_descriptions+=("Whether workflow has been installed")
+cmd_self_installed() {
+    local path_env="${path_root}/.env"
+    local path_env_dist="${path_root}/.env.example.dist"
```

Comment:
`.env.example.dist` needs to be `.env.example`. The latter is the standard name, certain tooling won't pick it up if it's named differently.